src/core/Modal/BaseModal/styles.ts

```diff
@@ -0,0 +1,45 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  modalContainer: (props: any) => ({
```

Comment:
Can we change props from `any` to either `Object` or `string` based on textAlign return type